src/server.js

```diff
@@ -5,7 +5,17 @@ const morgan = require('morgan');
 const app = express();
 const apiRouter = require('./resources/api.router');
 
+// Refer to https://enable-cors.org/server_expressjs.html
+const middlewareForAllowOrigin = (req, res, next) => {
```

Comment:
Middleware created by me.

more Info:  [Writing middleware for use in Express apps](http://expressjs.com/en/guide/writing-middleware.html)